go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/client9/misspell v0.3.4
 	github.com/go-lintpack/lintpack v0.5.2 // indirect
 	github.com/golangci/golangci-lint v1.31.0
-	github.com/google/addlicense v0.0.0-20200622132530-df58acafd6d5
+	github.com/google/addlicense v0.0.0-20200906110928-a0294312aa76
```

Comment:
@bogdandrutu are unrelated mod/sum changes eventually covered by dependabot or should they be included in standard PRs?